### DISCUSSION TITLE
[ModelEvento]: tipo di dato campi tipoModifica e genitore nell'oggetto ModelAnnotazioneModificativa

### DISCUSSION THREAD

**Discussion by siscomanpr** [italia/ansc#747] [Miglioramenti] [answered]
### Ambiente

pre-produzione
produzione

### Servizio

ModelEvento
R009 - Servizi cooperativi validazione evento

### idOperazione

assente

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

Serravalle Sesia

### Descrivi il problema

Si richiede che la definizione del tipo di dati dei campi `tipoModifica` e `genitore` dell'oggetto `ModelAnnotazioneModificativa` siano uniformati con il tipo di dato `integer` come già segnalato nella issues https://github.com/italia/ansc/issues/43

**Comment by mttfranci** [comment]
@siscomanpr  rispetto alla issue #43 nel frattempo è iniziata l'adozione controllata, quindi dobbiamo porre un pò più di cautela. Proviamo a verificare se tale modifica non crea problemi a nessuno. Anche in considerazione del fatto che il tipo number potenzialmente può contenere qualsiasi tipo di numero.

Saluti

Assistenza ANSC

**Comment by mttfranci** [comment, marked as answer]
@siscomanpr abbiamo condiviso all'allineamento settimanale di oggi di prenderci ancora una settimana per verificare se l'intervento eventualmente ha effetti collatorali sui client già implementati.
La settimana prossima, entro il 15, prenderemo una decisione definitiva.
Se qualcuno vuole aggiungere dei contributi può farlo in questa discussione.
Saluti

Assistenza ANSC
